Automattic/jetpack
# add get_themes to sync defaults

Follow up of #23884


This PR adds a new Callable to Sync that will send the list of installed Themes in the site

#### Changes proposed in this Pull Request:

* Add get_themes callable to Sync

#### Jetpack product discussion


1202064326332560-as-1202075586755890

#### Does this pull request change what data or activity we track or use?


This PR adds new information to the data we may sync about a site - depending on how a specific plugin configures Sync. It adds the list of installed themes.

#### Testing instructions:






* Sandbox your site
* Apply D78475-code to your sandbox
* Clear callables transient to make sure changes will be synced right away -> `delete_transient('jetpack_sync_callables_await');`
* Visit your site admin and make some changes.
* Verify that the `get_themes` callable was synced to the cache site
* See inline comments in #23884. Should we add a filter in get_themes function?
